src/NegativeFloat.js

```diff
@@ -0,0 +1,37 @@
+import { GraphQLScalarType } from 'graphql';
+import { GraphQLError } from 'graphql/error';
+import { Kind } from 'graphql/language';
+
+function processValue(value) {
+  if (isNaN(value)) {
+    throw new TypeError(`Value is not a number: ${value}`);
+  }
+
+  if (!(value < 0)) {
+    throw new TypeError(`Value is not a negative number: ${value}`);
+  }
+
+  return parseFloat(value);
+}
+
+export default new GraphQLScalarType({
+  name: 'PositiveFloat',
```

Comment:
I think this name was meant to be `NegativeFloat` ?

Reply:
Good catch.